qpc/utils.py

```diff
@@ -200,15 +249,32 @@ def read_server_config():
                 CONFIG_REQUIRE_TOKEN: require_token}
 
 
+def write_config(config_file_path, config_dict):
+    """Write configuration to config file.
+
+    :param config_dict: dict containing configuration info
+    :param config_file_path: path to configuration file
+    """
+    ensure_config_dir_exists()
+
+    with open(config_file_path, 'w') as configFile:
+        json.dump(config_dict, configFile, indent=4)
```

Comment:
Since both reading config functions go all the way to handle missing/invalid data, I'd make a small change here to make impossible saving key/value pairs with value in `(None, '')`. Could be a new function.

Pay attention to `configFile` as well. Could be `config_file`.

Reply:
Modified funtion read_insights_config(), mods are stated in above comments. 

And the snake ate the camel.